task_sdk/tests/execution_time/test_task_runner.py

```diff
@@ -86,3 +87,41 @@ def test_run_basic(test_dags_dir: Path):
 
     ti = parse(what)
     run(ti, log=mock.MagicMock())
+
+
+def test_run_deferred_basic(test_dags_dir: Path):
+    """Test running a basic task."""
+    from datetime import datetime
+
+    mock_utcnow = mock.MagicMock(return_value=timezone.datetime(2024, 11, 22, 0, 0, 0))
```

Comment:
We can use `timemachine` instead of mocking it

Reply:
updated in https://github.com/apache/airflow/pull/44241/commits/187c6eee9ef9076e6683b3c8088059eced916e2a